src/app/Scenes/MyAccount/MyAccountEditPhone.tsx

```diff
@@ -55,6 +63,28 @@ const MyAccountEditPhone: React.FC<{ me: MyAccountEditPhone_me$data }> = ({ me }
     }
   }
 
+  if (enableRedesignedSettings) {
+    return (
+      <MyProfileScreenWrapper title="Phone" onPress={handleSave} isValid={isValidNumber}>
+        <Flex
+          style={{
+            // We are setting a fixed height here to prevent the input from growing in height
+            // and pushing the save button when an error is present
+            height: INPUT_HEIGHT + 15,
```

Comment:
Have you tested this with font scaling? In case it breaks, we could change it to:

```suggestion
            // We are setting a fixed height here to prevent the input from growing in height
            // and pushing the save button when an error is present
            height: PixelRatio.getFontScale(INPUT_HEIGHT) + 15,
```

Reply:
good idea